packages/client/src/__tests__/tlog/verify/merkle.test.ts

```diff
@@ -181,4 +181,85 @@ describe('verifyMerkleInclusion', () => {
       );
     });
   });
+
+  describe('when all of the hashes are on the right side of the tree', () => {
```

Comment:
Adds a test case which specifically detects this scenario and will fail if the bit-length calculation is wrong.